builtin/common/vector.lua

```diff
@@ -18,40 +19,55 @@ function vector.equals(a, b)
 	       a.z == b.z
 end
 
+function vector.is_nil(a)
```

Comment:
no, a `nil` vector is the wrong word here I think

`is_zero` or `is_zero_vector` is probably the right name. I prefer `is_zero_vector` myself.

docs needed.